gcb/makestage/cloudbuild.yaml

```diff
@@ -1,8 +1,16 @@
-timeout: 14400s
+# This cloudbuild config file is intended to be triggered when a tag is pushed to the cert-manager repo
+# and will build a cert-manager release and push to Google Cloud Storage (GCS).
 
-steps:
+# The release won't be published automatically; this file just defines the build steps.
+
+# The full release and publish process is documented here:
+# https://cert-manager.io/docs/contributing/release-process/
 
-## Clone & checkout the cert-manager repository
+timeout: 2700s # 45m
```

Comment:
To match cert-manager

https://github.com/cert-manager/cert-manager/blob/f3b2a984482b36558b65a1511baffdccf38af616/gcb/build_cert_manager.yaml#L8-L10